### PR TITLE
Code fix

### PR DESCRIPTION
**Pull request by aberu78**

### CODE REVIEW

File: src/Main.java [hunk 1]
```diff
@@ -1,106 +1,192 @@
...
+        String str = "";
+        String bl = "";
+
+        while(!quit) {
+
+            instruction();
+            int inputNum = scanner.nextInt();
+            scanner.nextLine();
+
+            switch(inputNum){
```
Comment (roninsteven) [inline src/Main.java:24]: switch(scanner.nextInt()) I don't think the variable is required

File: src/Main.java [hunk 2]
```diff
@@ -125,48 +220,80 @@ public static int kapreKarsConstant(int num) {
...
-            // append a string into StringBuilder input1
             input1.append(String.valueOf(temp));
-            // reverse StringBuilder input1
             input1 = input1.reverse();
 
             int num2 = Integer.parseInt(String.valueOf(input1));
             System.out.println(num2 + " - " + num1 + " = " + (num2 - num1));
 
-            if (num2 - num1 == 6174) {
+            num = num2 - num1;
```
Comment (roninsteven) [inline src/Main.java:229]: Magic numbers please put a comment on what they are and what they mean

File: src/Main.java [hunk 3]
```diff
@@ -125,48 +220,80 @@ public static int kapreKarsConstant(int num) {
...
+            if (num == 6174) {
                 isValid = true;
             } else {
-                num = num2 - num1;
-                if (num < 1000)
+                if (num < 1000) {
```
Comment (roninsteven) [inline src/Main.java]: Same here

**Review by roninsteven — APPROVED** [review]
Great Job Kaori!!!

File: src/Main.java (hunk 1 above)
Comment (aberu78) [reply src/Main.java:24]: i put scanner.nextLine() to read newline  since nextInt() only read integer

Comment (roninsteven) [reply src/Main.java:24]: Ok

File: src/Main.java (hunk 3 above)
Comment (aberu78) [reply src/Main.java]: comment added next to the code. Fixed it. Thank you

File: src/Main.java (hunk 2 above)
Comment (aberu78) [reply src/Main.java:229]: comment added. fixed it. Thank you

**Pull request merged**